scripts/_common.sh

```diff
@@ -4,8 +4,10 @@
 # COMMON VARIABLES
 #=================================================
 
-# dependencies used by the app
-pkg_dependencies="deb1 deb2 php$YNH_DEFAULT_PHP_VERSION-deb1 php$YNH_DEFAULT_PHP_VERSION-deb2"
+NODEJS_VERSION=14
+GO_VERSION="1.18"
+
+pkg_dependencies="npm libc6-dev libssl-dev libxft-dev libhdf5-serial-dev libpng-dev libheif-examples librsvg2-bin libx264-dev libx265-dev libnss3 libfreetype6 libfreetype6-dev libfontconfig1 libfontconfig1-dev libzmq3-dev"
```

Comment:
You should consider using the NodeJS helper to install npm, in lieu of installing it on the whole system.

(log with failure to install npm: https://paste.yunohost.org/raw/akopihajig)